File: Discord.NET/noShards.cs
using System.Net.Http;
using System.Net.Http.Headers;
using System.Text;
using (var webclient = new HttpClient())
using (var content = new StringContent($"{{ \"count\": {client.Guilds.Count}}}", Encoding.UTF8, "application/x-www-form-urlencoded"))
{
    webclient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("token");
    HttpResponseMessage response = await webclient.PostAsync("https://discordbots.group/api/bot/YOUR_BOT_ID_HERE", content);
}

File: Discord.py/noShards.py
import discord
import asyncio
import aiohttp
import async_timeout
import json

dbg_token='your_token'

async def fetchPost(urlIn, d, h):
    async with aiohttp.ClientSession() as session:
        with async_timeout.timeout(10):
            async with session.post(urlIn, data=d, headers=h) as response:
                session.close()
                return response.status

await postToDBG():
    headers = {
        'Authorization': dbg_token,
        'Content-Type': 'application/x-www-form-urlencoded'
    }
    data = {"count": len(self.bot.guilds)}
    await fetchPost("https://discordbots.group/api/bot/{}".format(botID), data, headers)
